src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer.UnitTests/Utilities/ILspClient.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.LanguageServer.UnitTests;
+
+internal interface ILspClient
```

Comment:
is there much of a point in having this?  the test needs the actual lsp server as it needs to dispose of it